src/components/app/list/TriggerInfo.tsx

```diff
@@ -51,8 +53,20 @@ export class TriggerInfoModal extends Component<TriggerInfoModalProps, TriggerIn
             tagsEditable: false,
             hideImageTaggingHardDelete: false,
         }
+        this.commitInfoRef = React.createRef<HTMLDivElement>()
         this.selectMaterial = this.selectMaterial.bind(this)
         this.toggleChanges = this.toggleChanges.bind(this)
+        this.outsideClickHandler = this.outsideClickHandler.bind(this)
```

Comment:
this is not required in case of arrow function

Reply:
done